crates/circuit/src/dag_circuit.rs

```diff
@@ -5577,7 +5583,7 @@ impl DAGCircuit {
     /// Remove an operation node n.
     ///
     /// Add edges from predecessors to successors.
```

Comment:
You should change the docstring to reflect that the removed `PackedInstruction` is returned by the method.

Reply:
I updated the docstring in: https://github.com/Qiskit/qiskit/pull/13411/commits/03c5ba6c55869f4f1db6317a1c68f7b072716e8e